corehq/toggles.py

```diff
@@ -1973,3 +1973,10 @@ def _commtrackify(domain_name, toggle_is_enabled):
     TAG_INTERNAL,
     namespaces=[NAMESPACE_DOMAIN],
 )
+
+USH_WILDCARD_SEARCH = StaticToggle(
+    'ush_wildcard_search',
+    'COVID: Enable wildcard search in Advanced Case Search',
```

Comment:
what's the distinction between regular case search and "advanced" case search? I haven't heard that term before.

Reply:
May be I am the one that's calling it advanced. 

There is the case search box in standard reports that works off of regular case index. And then there is special case search ES backed search that's exposed by case-search API that's used by the mobile, which I am calling advanced. This is the feature that I am modifying.

Reply:
Yeah, I think previously anything to do with the "new" case-search index we just call "Case Search".